src/sonic-ctrmgrd/tests/ctrmgr_tools_test.py

```diff
@@ -0,0 +1,295 @@
+import imp
```

Comment:
In Python 3, `imp` is deprecated, superseded by `importlib`. Prefer `importlib`. Also please reorganize as per other similar comments.

Reply:
done